ghcide-test/data/multi-unit/a-1.0.0-inplace

```diff
@@ -16,6 +16,3 @@ base
 text
 -XHaskell98
 A
-+RTS
--A32M
--RTS
```

Comment:
These were intentionally added to test that we strip them correctly, why did you remove them?

Reply:
I see, when merging the master in, I forget to apply the patch from https://github.com/haskell/haskell-language-server/commit/c3b61feccbc87857390b9fdb542ce0b3a701d074, added back and revert the change.